docker-entrypoint.sh

```diff
@@ -18,7 +18,7 @@ if [ "$(id -u)" = "0" ]; then
         $TOMCATDIR/logs
 
     chown -R tomcat:tomcat $DHIS2HOME
-    exec su-exec tomcat "$0" "$@"
+    exec setpriv --reuid=tomcat --regid=tomcat --init-groups "$0" "$@"
```

Comment:
`setpriv` from `util-linux` offers the same functionality as `su-exec` but where `su-exec` is only in the main repo for alpine, `setpriv` is standard for Debian images.

Reply:
👍